sw/lib/python/settings_xml_parse.py

```diff
@@ -99,8 +99,8 @@ def __init__(self, ac_id):
                 if 'values' in the_setting.attrib:
                     settings.values = the_setting.attrib['values'].split('|')
                     count = int((settings.max_value - settings.min_value + settings.step) / settings.step)
-                    if len(settings.values) != count:
-                        print("Warning: possibly wrong number of values (%i) for %s (expected %i)" % (len(settings.values), shortname, count))
+#                     if len(settings.values) != count:
```

Comment:
why removing this warning ?

Reply:
sdlogger_download reads the stdout of this script and fails on reading this warning message.

Reply:
Fixed enum settings in module xml, restored warning message

Reply:
then, what about having a `verbose` (or `silence`) option for all the prints ?

Reply:
The warning print was actually correct, so I don't mind keeping it in place now that I fixed the error in the module xml. Alternatively, I would suggest printing warnings to stderr instead of adding a verbose flag?

Reply:
ok, I'm fine with a print to stderr

Reply:
Added printing to stderr, only tested with the sd download script.